### PR TITLE
Add transaction archive and low credit notification models to billing admin.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

File: go/billing/tests/test_admin.py [hunk 1]
```diff
@@ -38,7 +40,8 @@ def test_link_to_html_view(self):
         response = client.get(reverse('admin:billing_statement_changelist'))
         self.assertContains(response, "1 statement")
         self.assertContains(
-            response, '<a href="/billing/statement/%s">html</a>' % statement.id)
+            response,
+            '<a href="/billing/statement/%s">html</a>' % statement.id)
```
Comment (hodgestar) [inline go/billing/tests/test_admin.py]: This was a tiny pep8 clean up.

**Comment by hodgestar** [comment]
Ready for review.

File: go/billing/tests/test_admin.py (hunk 1 above)
Comment (justinvdm) [inline go/billing/tests/test_admin.py]: Looks good. Should we maybe use a singleton tuple as we usually do: `(statement.id,)`? Happy either way, but I remember that being the preferred convention.

**Comment by justinvdm** [comment]
:+1: when Travis is happy. Left a minor comment, but doesn't need a re-review.

File: go/billing/tests/test_admin.py (hunk 1 above)
Comment (hodgestar) [inline go/billing/tests/test_admin.py]: Might as well. Done. :)

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**